docs/api.rst

```diff
@@ -1793,9 +1793,8 @@ Translations
 
 .. http:get:: /api/translations/(string:project)/(string:component)/(string:language)/file/
 
-    Download current translation file as stored in VCS (without ``format``
-    parameter) or as converted to a standard format (currently supported:
-    Gettext PO, MO, XLIFF and TBX).
+    Download current translation file as stored in VCS (without the ``format``
```

Comment:
```suggestion
    Download current translation file as stored in the VCS (without the ``format``
```
alternative: "as it is stored". I think orangesunny might prefer that.

Reply:
You’re right :)